python_cross_site_scripting/PyCQA_flake8-bugbear/bugbear.py

```
@@ -0,0 +1,2497 @@
+            return
+
+        if call_path in B008_IMMUTABLE_CALLS | self.b008_b039_extend_immutable_calls:
+            self.generic_visit(node)
+            return
+
+        # Check if function call is actually a float infinity/NaN literal
+        if call_path == "float" and len(node.args) == 1:
+            try:
+                value = float(ast.literal_eval(node.args[0]))
+            except Exception:
+                pass
```

Ruff (0.13.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)